src/resources/views/crud/filters/select2_ajax.blade.php

```diff
@@ -6,7 +6,20 @@ class="nav-item dropdown {{ Request::get($filter->name)?'active':'' }}">
     <a href="#" class="nav-link dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">{{ $filter->label }} <span class="caret"></span></a>
     <div class="dropdown-menu p-0 ajax-select">
 	    <div class="form-group mb-0">
-	    	<select id="filter_{{ $filter->key }}" name="filter_{{ $filter->name }}" data-filter-key="{{ $filter->key }}" class="form-control input-sm select2" data-filter-type="select2_ajax" data-filter-name="{{ $filter->name }}" placeholder="{{ $filter->placeholder }}">
+            <select
+                id="filter_{{ $filter->key }}"
+                name="filter_{{ $filter->name }}"
+                class="form-control input-sm select2"
+                placeholder="{{ $filter->placeholder }}"
+                data-filter-key="{{ $filter->key }}"
+                data-filter-type="select2_ajax"
+                data-filter-name="{{ $filter->name }}"
+                data-select-key="{{ $filter->options['select_key'] ?? 'id' }}"
+                data-select-attribute="{{ $filter->options['select_attribute'] ?? 'name' }}"
+                filter-minimum-input-length="{{ $filter->options['minimum_input_length'] ?? 2 }}"
+                filter-method="{{ $filter->options['method'] ?? 'GET' }}"
+                filter-quiet-time="{{ $filter->options['quiet_time'] ?? 500 }}"
```

Comment:
@pxpm Just noticed now, `quietMillis` was 50, and now it defaults to 500.
Should we keep it in 50? to avoid BC?

Reply:
```suggestion
                filter-quiet-time="{{ $filter->options['quiet_time'] ?? 50 }}"
```

Reply:
We use 500 in other ajax fields.

Technically it's a breaking change, but pratically I don't think it's a BC. This small difference is pretty much unoticed by the low difference between values. 

We already use `500` as a `delay` in other ajax fields, so I changed to 500 here to make it similar to the others. 

I would keep `500` here, because of what I already said, the difference unoticeable. 

Let me know what you think.

Best,
Pedro

Reply:
I agree!